@xen-orchestra/backups/_runners/_writers/_MixinRemoteWriter.mjs

```diff
@@ -70,7 +70,7 @@ export const MixinRemoteWriter = (BaseClass = Object) =>
 
         await handler.outputFile(taskFile, this._vmUuid)
         const remotePath = handler.getRealPath()
-        await MergeWorker.run(remotePath)
+        await MergeWorker.run(remotePath, mergeWorkerConcurrency)
```

Comment:
I'm not fond of this, because the worker is an independent process which might already be running (or even running on another machine), in that case, the new concurrency will not be taken into account. :thinking: 

I think an ideal solution would be to use `app-conf` for the worker and set a config file somewhere on the remote, but that may be too complicated for now.

What's your opinion on this?

Reply:
That seems overcomplicated , at least as long as we don't have an understanding of the real word use of independant merge worker